gnovm/cmd/gno/util.go

```diff
@@ -100,6 +100,64 @@ func gnoPackagesFromArgs(args []string) ([]string, error) {
 	return paths, nil
 }
 
+func gnoPackagePathsFromPattern(patterns []string) ([]string, error) {
+	paths := []string{}
+	recursiveLookup := false
+	for _, p := range patterns {
+		// TODO:
+		//     - Support more patterns
+		//     - Move this logic to separate helper function?
+		if strings.HasSuffix(p, "/...") {
+			recursiveLookup = true
+			p = strings.TrimSuffix(p, "/...")
+		}
+
```

Comment:
What do you think about extracting this logic into a new method (ex. `parsePattern`), that returns a flag indicating if the lookup is recursive, and the pattenr itself (trimmed if so, base pattern if not)

Reply:
In later PRs. See comment.